resources/views/auth/validate2fa.blade.php

```diff
@@ -3,7 +3,7 @@
 @section('content')
     <div class="container">
       <form action="{{ session('oauth') ? route('oauth.validate2fa') : route('validate2fa') }}" method="post">
-        <input type="hidden" name="url" value="{{ urlencode(url()->current()) }}" />
+        <input type="hidden" name="url" value="dashboard" />
```

Comment:
I would rather use a `route` url
```suggestion
        <input type="hidden" name="url" value="{{ urlencode(route('dashboard.index')) }}" />
```